pkg/streamingpromql/operators/binops/vector_scalar_binary_operation.go

```diff
@@ -48,7 +48,7 @@ func NewVectorScalarBinaryOperation(
 	memoryConsumptionTracker *limiter.MemoryConsumptionTracker,
 	annotations *annotations.Annotations,
 	expressionPosition posrange.PositionRange,
-) (*VectorScalarBinaryOperation, error) {
+) (types.InstantVectorOperator, error) {
```

Comment:
Why has this changed?

Reply:
Thanks for noticing, it's a leftover from the initial implementation, I'll clean this up.